frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.ts

```diff
@@ -105,8 +111,12 @@ export const useValuesForQuickStartContext = (): QuickStartContextValues => {
         ...qs,
         [quickStartId]: getDefaultQuickStartState(totalTasks, QuickStartStatus.IN_PROGRESS),
       }));
+      fireTelemetryEvent('Quick Start Initiated', {
+        id: quickStartId,
+        type: 'restart',
```

Comment:
Add quick start `name`

Reply:
After investigation, the adding the name at this point would require some api refactoring and not worth the effort for the initial telemetry release.